src/main/java/wooteco/subway/controller/GlobalExceptionHandler.java

```diff
@@ -18,4 +17,10 @@
         logger.error(String.valueOf(e.getBody()));
         return ResponseEntity.status(e.getHttpStatus()).body(e.getBody());
     }
+
+    @ExceptionHandler(Exception.class)
+    public ResponseEntity<Object> handleUnchecked(Exception e) {
+        logger.error(String.valueOf(e.getMessage()));
+        return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body(e.getMessage());
```

Comment:
개발자가 정의하지 않는 오류의 메세지를 그대로 내보내는 경우 어플리케이션 구조나 사용 기술등이 외부에 노출 될 수 있어요.
보통 에러메세지는 로그로 남기고 외부로는 "서버 오류 발생" 같은 메세지 정도만 내보내주고 있어요. 

```suggestion
        logger.error("오류 발생 >>> {}", e.getMessage());
        return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body("서버 오류 발생");
```